Tests/Admin/AdminTest.php

```diff
@@ -1524,18 +1524,6 @@ public function testGetSubject()
         $admin->setModelManager($modelManager);
 
         $admin->setRequest(new Request(array('id' => 'azerty')));
-        $this->assertFalse($admin->getSubject());
-
-        $admin->setSubject(null);
-        $admin->setRequest(new Request(array('id' => 42)));
```

Comment:
@Soullivaneuh I think he should delete them : the behavior is changed, there is no validity check to test anymore.